apis/cloudsearch/v1alpha1/generator-config.yaml

```diff
@@ -0,0 +1,2 @@
+ignore:
```

Comment:
can you check `Move versioned generator configs to API group folders` https://github.com/crossplane/provider-aws/pull/1096 ?

Reply:
Done!